src/features/applicant-details.feature

```diff
@@ -0,0 +1,48 @@
+@regression
+Feature: BNGP-179, BNGP-1099 Site Provider applicant details
+
+    As a Site Provider
+    I need to provide my personal details
+    So that the Operator knows who is applying to register
+
+    Background: Start the journey and add user details
+        Given I navigate to the "start" page
+        And I start my registration
+        And I confirm my "fullname" as "BNGP-1469"
+        And I confirm my role as a "landowner"
+        And I confirm my "email address" as "me@you.com"
```

Comment:
if you use example.com as an email domain you never have to worry about spamming an actual email inbox